src/App.js

```diff
@@ -30,7 +30,7 @@ class App extends React.Component<Props, State> {
           <MenuTrigger onClick={() => this.toggleMenu(true)} />
         </Header>
         {this.state.showMenu && (
```

Comment:
Since react-portal now takes care of rendering the inner portal based on `isOpen`, it shouldn't need this `showMenu` check that wraps it. Could you remove this wrapping expression?